app/src/main/java/space/taran/arknavigator/mvp/presenter/adapter/ResourcesGridPresenter.kt

```diff
@@ -51,6 +53,9 @@ class ResourcesGridPresenter(
     fun getCount() = selection.size
 
     fun bindView(view: FileItemView) {
+        if (!resourcesReady)
```

Comment:
is it a better idea to have this isResourceReady to be dependent on something else - not based on which method from the presenter is actually being called. For example to be dependant of some state(if something has finished or not)?